packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -1,77 +1,50 @@
 # template: hammer_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
 %global gem_name hammer_cli_katello
 %global plugin_name katello
 
-%global release 1
-%global prereleasesource pre.master
-%global prerelease %{?prereleasesource:.}%{?prereleasesource}
```

Comment:
I think you want to keep *these* to allow it to be nightly built

Reply:
Yeah I just realized that, I'll fix the naming.